### PR TITLE
fix building in VS with PDB conversions

### PR DESCRIPTION
**Pull request by brettfo**
There were two issues with building in Visual Studio:

1.  Converting the portable PDB to a Windows-style PDB was failing because the command-line build explicitly sets the `%NUGET_PACKAGES%` environment variable to `<repo-root>\packages`, but that variable wasn't set when `VisualFSharp.sln` was open in VS so it was falling back to the common location of `%USERPROFILE%\.nuget\packages` which doesn't work.  The fix for that is to set the fallback to `<repo-root>\packages`.
2.  The property pages project explicitly sets its `$(OutputPath)` variable which means the `HACK_CopyOutputsToTheProperLocation` target was being unnecessarily run which was resulting in incorrect PDBs and locked files.

### CODE REVIEW

File: vsintegration/src/FSharp.ProjectSystem.PropertyPages/FSharp.PropertiesPages.vbproj [hunk 1]
```diff
@@ -34,6 +34,7 @@
     <GeneratePkgDefFile>true</GeneratePkgDefFile>
     <IncludePkgdefInVSIXContainer>true</IncludePkgdefInVSIXContainer>
     <UsePackageTargetFallbackHack>true</UsePackageTargetFallbackHack>
+    <DisableOutputPathCopying>true</DisableOutputPathCopying>
```
Comment (dsyme) [inline vsintegration/src/FSharp.ProjectSystem.PropertyPages/FSharp.PropertiesPages.vbproj]: We might need this on a few more projects

Comment (brettfo) [reply vsintegration/src/FSharp.ProjectSystem.PropertyPages/FSharp.PropertiesPages.vbproj]: At this point we shouldn't.  The only projects picking up the path copying hack from `FSharp.Directory.Build.targets` are projects converted to the SDK where the copy hack is necessary, and this property pages project.  All other projects in our repo directly write outputs to `release\net40\bin`, but because we ultimately depend on `$(TargetFramework)` from SDK projects to determine `release\net40\bin` vs. `release\coreclr\bin`, we have to copy files after the build has completed.

tl;dr - This is the only project file that can properly set `$(OutputPath)` _and_ that pulls in `FSharp.Directory.Build.targets` which defaults to the copy hack.

Comment (dsyme) [reply vsintegration/src/FSharp.ProjectSystem.PropertyPages/FSharp.PropertiesPages.vbproj]: OK, understood, thanks.  I thought it was about the Copy File conflicts.
